piksi_tools/console/tracking_view.py

```diff
@@ -88,9 +91,9 @@ def tracking_state_callback(self, sbp_msg, **metadata):
     for n, k in enumerate(self.states):
       s = sbp_msg.states[n]
       prn = s.sid.sat
-      if (s.sid.code == 0):
+      if (s.sid.code == 0 or s.sid.code == 1):
```

Comment:
This was correct as is. GPS L1 is encoded as prn-1 over SBP for backwards compatibility. Other code types are not converted.